boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -137,8 +137,18 @@
 #endif
 
 #ifdef CONFIG_BOOT_DECOMPRESSION
+
```

Comment:
```suggestion
```

Reply:
Moved after the ifdef, to separate blocks.